lib/cli/cliPlugin.js

```diff
@@ -0,0 +1,346 @@
+                    const enabled = pluginEnabled(
+                        pluginName,
+                        await objects.getObjectAsync('system.config'),
+                        instance
+                    );
+                    CLI.success.pluginStatus(
+                        pluginName,
+                        hostname,
+                        instance,
+                        enabled
+                    );
+                    return void callback();
+                } catch (err) {
+                    CLI.error.unknown(err);
+                    return void callback(1);
+                }
+            }
+        );
+    }
+};
+
+/**
+ * Checks if a plugin exists and can be configured
+ * @param {string} pluginName
+ * @param {Record<string, any>} systemConfig
+ * @param {string} [adapter] (optional) - If passed, the adapter configuration will be searched for defined plugins instead of js-controller
+ */
+function pluginExists(pluginName, systemConfig, adapter) {
+    // 1. check if the plugin is defined in io-package.json
+    // TODO: replace this with fs-extra methods #799
+    try {
+        const ioPackPath = adapter
+            ? path.join(tools.getAdapterDir(adapter), 'io-package.json')
+            : controllerIoPackPath;
+        const ioPack = JSON.parse(fs.readFileSync(ioPackPath, 'utf8'));
+        if (
+            ioPack &&
+            ioPack.common &&
+            ioPack.common.plugins &&
+            pluginName in ioPack.common.plugins
+        ) {
+            return true;
+        }
+    } catch {
+        /* ignore */
+    }
+
+    // 2. check if the plugin is defined in iobroker.json
+    return (
+        systemConfig &&
+        systemConfig.plugins &&
+        pluginName in systemConfig.plugins
+    );
+}
+
+/**
+ * Checks if a plugin exists and can be configured
+ * @param {string} pluginName
+ * @param {Record<string, any>} systemConfig
+ * @param {string} [adapter] (optional) - If passed, the adapter configuration will be searched for defined plugins instead of js-controller
+ */
+function pluginEnabled(pluginName, systemConfig, adapter) {
+    // 1. check if diagnostics were disabled in ioBroker
+    if (systemConfig && systemConfig.common && systemConfig.common.diag === 'none') {
+        return false;
+    }
+
+    // 2. check if the plugin is disabled in io-package.json
+    // TODO: replace this with fs-extra methods #799
+    try {
+        const ioPackPath = adapter
+            ? path.join(tools.getAdapterDir(adapter), 'io-package.json')
+            : controllerIoPackPath;
+        const ioPack = JSON.parse(fs.readFileSync(ioPackPath, 'utf8'));
+        if (
+            ioPack &&
+            ioPack.common &&
+            ioPack.common.plugins &&
+            pluginName in ioPack.common.plugins
+        ) {
+            return ioPack.common.plugins[pluginName].enabled !== false;
+        }
+    } catch {
+        /* ignore */
+    }
+
+    // default: enabled
```

Comment:
check for enabled in systemConfig is missing here ...

Reply:
yep and the check was not correct, fixed.